src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +12,25 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+
+    private static final String FIRST_CHAR = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        String fileText = "";
+        try (FileReader fileReader = new FileReader(fileName)) {
+            int oneLetter;
+            while ((oneLetter = fileReader.read()) != -1) {
+                fileText += (char) oneLetter;
+            }
```

Comment:
let's use BufferedReader so you will read the whole line per iteration instead of single character